### ISSUE TITLE
6.0 Feedback:  Module actions created during run pause do not happen immediately.  They seem to happen once the protocol is unpaused.

### ISSUE THREAD

**Issue by y3rsh** [Opentrons/opentrons#10647]
### Overview

At first I thought the commands were ignored. BUT once I unpaused they all seemed to happen (heard fans spin up and mag module engage)

https://user-images.githubusercontent.com/502770/172444072-8412d8e1-10a4-4881-9f94-8310a9fa44ff.mp4

> Note that module controls work as expected before protocol run is started.

### Current Behavior

Module commands happen when unpaused.

### Expected Behavior

Module commands happen immediately.

### Steps To Reproduce

1. Have modules attached to the robot for a protocol like below.
2. Run the protocol
3. Pause the protocol
4. Go to modules tab
5. Interact with the modules
6. nothing happens
7. Unpause
8. All the interactions you created happen.

### Operating system

Mac

### Robot setup or anything else?

[unused_module.py.zip](https://github.com/Opentrons/opentrons/files/8855249/unused_module.py.zip)

**Comment by mcous** [comment]
What does the HTTP API report for the module control commands while the protocol is paused? Is the first command in a `running` state or is it in a `queued` state?

**Comment by y3rsh** [comment]
@mcous 
I paused the protocol
Then issued the disengage from the mag modules card 
when I get the /runs/RUN_ID/commands
```js
{
    'data': [
        {
            'id': 'commands.LOAD_LABWARE-0',
            'key': 'commands.LOAD_LABWARE-0',
            'commandType': 'loadLabware',
            'createdAt': '2022-06-07T20:40:16.671560+00:00',
            'startedAt': '2022-06-07T20:40:16.671560+00:00',
            'completedAt': '2022-06-07T20:40:16.671560+00:00',
            'status': 'succeeded',
            'params': {
                'location': {'slotName': '1'},
                'loadName': 'opentrons_96_tiprack_300ul',
                'namespace': 'opentrons',
                'version': 1,
                'displayName': '300ul tips'
            }
        },
        {
            'id': 'commands.LOAD_LABWARE-1',
            'key': 'commands.LOAD_LABWARE-1',
            'commandType': 'loadLabware',
            'createdAt': '2022-06-07T20:40:16.861816+00:00',
            'startedAt': '2022-06-07T20:40:16.861816+00:00',
            'completedAt': '2022-06-07T20:40:16.861816+00:00',
            'status': 'succeeded',
            'params': {
                'location': {'slotName': '5'},
                'loadName': 'opentrons_96_tiprack_20ul',
                'namespace': 'opentrons',
                'version': 1,
                'displayName': '20ul tips'
            }
        },
        {
            'id': 'commands.LOAD_PIPETTE-0',
            'key': 'commands.LOAD_PIPETTE-0',
            'commandType': 'loadPipette',
            'createdAt': '2022-06-07T20:40:19.019832+00:00',
            'startedAt': '2022-06-07T20:40:19.019832+00:00',
            'completedAt': '2022-06-07T20:40:19.019832+00:00',
            'status': 'succeeded',
            'params': {'pipetteName': 'p20_single_gen2', 'mount': 'right'}
        },
        {
            'id': 'commands.LOAD_PIPETTE-1',
            'key': 'commands.LOAD_PIPETTE-1',
            'commandType': 'loadPipette',
            'createdAt': '2022-06-07T20:40:21.162072+00:00',
            'startedAt': '2022-06-07T20:40:21.162072+00:00',
            'completedAt': '2022-06-07T20:40:21.162072+00:00',
            'status': 'succeeded',
            'params': {'pipetteName': 'p300_multi_gen2', 'mount': 'left'}
        },
        {
            'id': 'commands.LOAD_MODULE-0',
            'key': 'commands.LOAD_MODULE-0',
            'commandType': 'loadModule',
            'createdAt': '2022-06-07T20:40:21.250903+00:00',
            'startedAt': '2022-06-07T20:40:21.250903+00:00',
            'completedAt': '2022-06-07T20:40:21.250903+00:00',
            'status': 'succeeded',
            'params': {
                'model': 'magneticModuleV2',
                'location': {'slotName': '9'},
                'moduleId': 'module-0'
            }
        },
        {
            'id': 'commands.LOAD_MODULE-1',
            'key': 'commands.LOAD_MODULE-1',
            'commandType': 'loadModule',
            'createdAt': '2022-06-07T20:40:21.347332+00:00',
            'startedAt': '2022-06-07T20:40:21.347332+00:00',
            'completedAt': '2022-06-07T20:40:21.347332+00:00',
            'status': 'succeeded',
            'params': {
                'model': 'temperatureModuleV1',
                'location': {'slotName': '4'},
                'moduleId': 'module-1'
            }
        },
        {
            'id': 'commands.LOAD_MODULE-2',
            'key': 'commands.LOAD_MODULE-2',
            'commandType': 'loadModule',
            'createdAt': '2022-06-07T20:40:21.517900+00:00',
            'startedAt': '2022-06-07T20:40:21.517900+00:00',
            'completedAt': '2022-06-07T20:40:21.517900+00:00',
            'status': 'succeeded',
            'params': {
                'model': 'thermocyclerModuleV1',
                'location': {'slotName': '7'},
                'moduleId': 'module-2'
            }
        },
        {
            'id': 'commands.LOAD_LABWARE-2',
            'key': 'commands.LOAD_LABWARE-2',
            'commandType': 'loadLabware',
            'createdAt': '2022-06-07T20:40:21.592310+00:00',
            'startedAt': '2022-06-07T20:40:21.592310+00:00',
            'completedAt': '2022-06-07T20:40:21.592310+00:00',
            'status': 'succeeded',
            'params': {
                'location': {'moduleId': 'module-1'},
                'loadName': 'opentrons_96_aluminumblock_nest_wellplate_100ul',
                'namespace': 'opentrons',
                'version': 1,
                'displayName': 'Temperature-Controlled plate'
            }
        },
        {
            'id': 'commands.LOAD_LABWARE-3',
            'key': 'commands.LOAD_LABWARE-3',
            'commandType': 'loadLabware',
            'createdAt': '2022-06-07T20:40:21.752136+00:00',
            'startedAt': '2022-06-07T20:40:21.752136+00:00',
            'completedAt': '2022-06-07T20:40:21.752136+00:00',
            'status': 'succeeded',
            'params': {
                'location': {'moduleId': 'module-0'},
                'loadName': 'nest_96_wellplate_100ul_pcr_full_skirt',
                'namespace': 'opentrons',
                'version': 1
            }
        },
        {
            'id': 'commands.LOAD_LABWARE-4',
            'key': 'commands.LOAD_LABWARE-4',
            'commandType': 'loadLabware',
            'createdAt': '2022-06-07T20:40:21.968088+00:00',
            'startedAt': '2022-06-07T20:40:21.968088+00:00',
            'completedAt': '2022-06-07T20:40:21.968088+00:00',
            'status': 'succeeded',
            'params': {
                'location': {'moduleId': 'module-2'},
                'loadName': 'nest_96_wellplate_100ul_pcr_full_skirt',
                'namespace': 'opentrons',
                'version': 1
            }
        },
        {
            'id': 'commands.LOAD_LABWARE-5',
            'key': 'commands.LOAD_LABWARE-5',
            'commandType': 'loadLabware',
            'createdAt': '2022-06-07T20:40:22.075494+00:00',
            'startedAt': '2022-06-07T20:40:22.075494+00:00',
            'completedAt': '2022-06-07T20:40:22.075494+00:00',
            'status': 'succeeded',
            'params': {
                'location': {'slotName': '6'},
                'loadName': 'cpx_4_tuberack_100ul',
                'namespace': 'custom_beta',
                'version': 1,
                'displayName': '4 tubes'
            }
        },
        {
            'id': 'commands.LOAD_LABWARE-6',
            'key': 'commands.LOAD_LABWARE-6',
            'commandType': 'loadLabware',
            'createdAt': '2022-06-07T20:40:22.150685+00:00',
            'startedAt': '2022-06-07T20:40:22.150685+00:00',
            'completedAt': '2022-06-07T20:40:22.150685+00:00',
            'status': 'succeeded',
            'params': {
                'location': {'slotName': '3'},
                'loadName': 'nest_96_wellplate_100ul_pcr_full_skirt',
                'namespace': 'opentrons',
                'version': 1,
                'displayName': 'logo destination'
            }
        },
        {
            'id': 'commands.LOAD_LABWARE-7',
            'key': 'commands.LOAD_LABWARE-7',
            'commandType': 'loadLabware',
            'createdAt': '2022-06-07T20:40:22.286347+00:00',
            'startedAt': '2022-06-07T20:40:22.286347+00:00',
            'completedAt': '2022-06-07T20:40:22.286347+00:00',
            'status': 'succeeded',
            'params': {
                'location': {'slotName': '2'},
                'loadName': 'nest_12_reservoir_15ml',
                'namespace': 'opentrons',
                'version': 1,
                'displayName': 'dye container'
            }
        },
        {
            'id': 'command.PICK_UP_TIP-0',
            'key': 'command.PICK_UP_TIP-0',
            'commandType': 'pickUpTip',
            'createdAt': '2022-06-07T20:40:22.336659+00:00',
            'startedAt': '2022-06-07T20:40:22.336659+00:00',
            'completedAt': '2022-06-07T20:40:29.133831+00:00',
            'status': 'succeeded',
            'params': {
                'pipetteId': 'pipette-0',
                'labwareId': 'labware-1',
                'wellName': 'A1',
                'wellLocation': {
                    'origin': 'top',
                    'offset': {'x': 0, 'y': 0, 'z': 0}
                }
            }
        },
        {
            'id': '332a6e9e-4aec-418d-9b79-9c36b320f967',
            'key': '332a6e9e-4aec-418d-9b79-9c36b320f967',
            'commandType': 'loadModule',
            'createdAt': '2022-06-07T20:40:36.781926+00:00',
            'startedAt': '2022-06-07T20:40:36.802288+00:00',
            'completedAt': '2022-06-07T20:40:36.835887+00:00',
            'status': 'succeeded',
            'params': {
                'model': 'magneticModuleV2',
                'location': {'slotName': '9'},
                'moduleId': 'module-0'
            },
            'intent': 'setup'
        },
        {
            'id': 'ad80e56a-222b-4a5f-b878-80b6c838e37e',
            'key': 'ad80e56a-222b-4a5f-b878-80b6c838e37e',
            'commandType': 'loadModule',
            'createdAt': '2022-06-07T20:40:36.789462+00:00',
            'startedAt': '2022-06-07T20:40:36.837246+00:00',
            'completedAt': '2022-06-07T20:40:36.870141+00:00',
            'status': 'succeeded',
            'params': {
                'model': 'temperatureModuleV1',
                'location': {'slotName': '4'},
                'moduleId': 'module-1'
            },
            'intent': 'setup'
        },
        {
            'id': '2e1f6a4e-db94-405f-a5d1-e79cffa5a6df',
            'key': '2e1f6a4e-db94-405f-a5d1-e79cffa5a6df',
            'commandType': 'loadModule',
            'createdAt': '2022-06-07T20:40:36.796869+00:00',
            'startedAt': '2022-06-07T20:40:36.871530+00:00',
            'completedAt': '2022-06-07T20:40:36.895270+00:00',
            'status': 'succeeded',
            'params': {
                'model': 'thermocyclerModuleV1',
                'location': {'slotName': '7'},
                'moduleId': 'module-2'
            },
            'intent': 'setup'
        },
        {
            'id': '3265784f-b51c-4525-80dd-beb451f695c6',
            'key': '3265784f-b51c-4525-80dd-beb451f695c6',
            'commandType': 'magneticModule/disengage',
            'createdAt': '2022-06-07T20:40:41.003270+00:00',
            'startedAt': '2022-06-07T20:40:41.005115+00:00',
            'status': 'running',
            'params': {'moduleId': 'module-0'},
            'intent': 'setup'
        }
    ],
    'links': {
        'current': {
            'href': '/runs/61ddedbc-6690-4c6d-ba36-f13d7ad328dd/commands/3265784f
-b51c-4525-80dd-beb451f695c6',
            'meta': {
                'runId': '61ddedbc-6690-4c6d-ba36-f13d7ad328dd',
                'commandId': '3265784f-b51c-4525-80dd-beb451f695c6',
                'index': 17,
                'key': '3265784f-b51c-4525-80dd-beb451f695c6',
                'createdAt': '2022-06-07T20:40:41.003270+00:00'
            }
        }
    },
    'meta': {'cursor': 0, 'totalLength': 18}
}
```

**Comment by mcous** [comment]
Closed by #10865 and #10675

**Comment by y3rsh** [comment]
verified, module controls are not available during run paused or running.
